packages/nodes/link/src/components/FloatingLink/OpenLinkButton.tsx

```diff
@@ -15,13 +16,15 @@ export const useOpenLinkButton = (
   props: HTMLPropsAs<'a'>
 ): HTMLPropsAs<'a'> => {
   const editor = useEditorRef();
+  const selection = usePlateSelection();
 
   const entry = useMemo(
     () =>
       findNode<TLinkElement>(editor, {
         match: { type: getPluginType(editor, ELEMENT_LINK) },
       }),
-    [editor]
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Instead of this comment maybe it make sense to pass selection to the `findNode`? Or it ok as it is?